hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -588,7 +588,7 @@
           desc: '''
                 This is the window size for all health tests.
                 This value is used when entropy is being tested in FIPS/CC compliance mode (for simplicity referred to as FIPS mode).
-                The default value is (2048 bits * 1 clock/4 bits);
+                The default value is 2048 bits;
 
                 Note that the number of tested bits taken by the conditioner to produce a seed is equal to the window size x 4.
                 The only exception is the startup seed which is produced using the bits of two subsequent windows, i.e., 2 x window size x 4 tested bits.
```

Comment:
These two sentences and the one after them need to be adapted as well w.r.t. the factor 4, right?